File: documentation-examples-scala/src/scala/dev/typr/foundations/docs/db2/IntegerTypes.scala
package dev.typr.foundations.docs.db2
import dev.typr.foundationssc.*
import dev.typr.foundationssc.data.*



@SuppressWarnings(Array("unused"))
object IntegerTypes:
  //start
  val smallType: Db2Type[Short] = Db2Types.smallint
  val intType: Db2Type[Int] = Db2Types.integer
  val bigType: Db2Type[Long] = Db2Types.bigint
  //stop

File: documentation-examples-kotlin/src/kotlin/dev/typr/foundations/docs/db2/IntegerTypes.kt
package dev.typr.foundations.docs.db2

import dev.typr.kotlinfoundations.*
import dev.typr.kotlinfoundations.data.*

@Suppress("unused")
class IntegerTypes {
    //start
    val smallType: Db2Type<Short> = Db2Types.smallint
    val intType: Db2Type<Int> = Db2Types.integer
    val bigType: Db2Type<Long> = Db2Types.bigint
    //stop
}
